src/OpenTK.Windowing.Desktop/NativeWindowSettings.cs

```diff
@@ -0,0 +1,37 @@
+using System.Drawing;
+using Humanizer;
+using Humanizer.Localisation.Formatters;
+using OpenToolkit.GraphicsLibraryFramework;
+using OpenToolkit.Windowing.Common;
+
+namespace OpenToolkit.Windowing.Desktop
+{
+    public class NativeWindowSettings : INativeWindowProperties
+    {
+        /// <summary>
+        /// Gets the default settings for a NativeWinow.
+        /// </summary>
+        public static readonly NativeWindowSettings Default = new NativeWindowSettings();
+
+        public unsafe Monitor* CurrentMonitor { get; set; } = GLFWProvider.GLFW.Value.GetPrimaryMonitor();
```

Comment:
Is there a reason why CurrentMonitor is not part of INativeWindowProperties?
https://github.com/opentk/opentk/pull/867/files#diff-1534d749d75f06fba6a77a9d50e7420cR108
then we could forward INativeWindowProperties instead of NativeWindowSettings, which would make it consistent with the already passed IGameWindowProperties...

Reply:
This sounds good on paper, but I'd like to hear some second opinions first. Does anyone have any complaints about this?

Reply:
Fixed.